Stable-Branch-Strategy.md

```diff
@@ -0,0 +1,103 @@
+Branch and release maintenance for Kata Containers project
+
+# Overview
+
+As detailed in the [release documentation](https://github.com/kata-containers/documentation/blob/master/Releases.md),
+the Kata Containers project makes use of semantic versioning.  A release version is described
+by MAJOR.MINOR.PATCH.  Early in the project, we will introduce many new features, which require
+minor version changes as well as bug fixes.  To facilitate a stable user environment, Kata will
+begin to provide stable branch-based releases and a master branch release.
+
+## Stable branch patch criteria
+
+No new features should be introduced to stable branches.  This is intended to limit risk to users,
+providing only bug and security fixes.
+
+## Branch Management
+Kata Containers will maintain two stable release branches in addition to the master branch.
+Once a new MAJOR or MINOR release is created from master, a new stable branch is created for
+the prior MAJOR or MINOR release and the older stable branch is no longer maintained.
```

Comment:
I think we need to explain the options for users currently using the older stable branch:

- How do they know the branch has become EOL? (ML posting)
- How do they upgrade to the newer stable branch? (automatic via OBS distro-packaged components?)

We should explain how users can establish the version they are currently running (by running `kata-runtime kata-env`).

Reply:
Addressed the easy parts of this feedback.  I think the end-user OBS usage is still in flux, and I am not sure if it should be included in this document or an end-user installation document.